### PR TITLE
Golden file changes for 2020_10_09 release

### PR DESCRIPTION
**Pull request by tjann**

### CODE REVIEW

File: golden_response/staging/get_landing_page_data/asm.json [hunk 1]
```diff
@@ -655,136 +651,8 @@
       "WorldBank_SP_DYN_IMRT_FE_IN": {},
       "WorldBank_SP_DYN_IMRT_IN": {},
       "WorldBank_SP_DYN_IMRT_MA_IN": {},
-      "WorldBank_SP_RUR_TOTL": {
```
Comment (tjann) [inline golden_response/staging/get_landing_page_data/asm.json:658]: This is expected. After updating the chart config in g3, these should return under human readable names.

File: golden_response/staging/get_stat_ranking/california.json [hunk 2]
```diff
@@ -5,297 +5,292 @@
         "info": [
           {
             "rank": 1,
-            "value": 10039107,
+            "value": 10170292,
```
Comment (tjann) [inline golden_response/staging/get_stat_ranking/california.json:8]: Did we switch Census surveys?

File: golden_response/staging/get_stat_ranking/country.json [hunk 3]
```diff
@@ -5,7 +5,7 @@
         "info": [
           {
             "rank": 1,
-            "value": 1396530000,
+            "value": 1409517397,
```
Comment (tjann) [inline golden_response/staging/get_stat_ranking/country.json:8]: I guess there is no way to see the source?

File: golden_response/staging/get_stat_ranking/crime_percapita.json [hunk 4]
```diff
@@ -10,1383 +10,1383 @@
           },
           {
             "rank": 2,
+            "value": 60960,
```
Comment (tjann) [inline golden_response/staging/get_stat_ranking/crime_percapita.json:13]: My guess is this geo might have changed typeOfs. shanth?

Comment (pradh) [reply golden_response/staging/get_stat_ranking/crime_percapita.json:13]: I'll take a look at this later tonight (filed b/170684975)..

Comment (pradh) [reply golden_response/staging/get_stat_ranking/crime_percapita.json:13]: It might be easier to debug if we push to staging, to detect other bugs.

**Review by pradh — APPROVED** [review]

**Pull request merged**